hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/RestrictedPatientSearchService.kt

```diff
@@ -66,8 +68,8 @@ class RestrictedPatientSearchService(
     return query?.let {
       val searchSourceBuilder = SearchSourceBuilder().apply {
         query(query.withDefaults(searchCriteria))
-        size(pageable.pageSize)
-        from(pageable.offset.toInt())
+        size(pageSize)
+        from(pageOffset)
```

Comment:
note that the sort order is `prisonerNumber` which means that the sorting should be stable.  It is sorting by `score` that can mean that different shards return different data.